MapboxCoreNavigation/String.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
```

Comment:
You should do something like:
```swift
#if os(iOS)
import UIKit
#elseif os(watchOS)
import WatchKit
```

However, that could be tail work or part of https://github.com/mapbox/mapbox-navigation-ios/issues/305